src/Tokenizer/Tokens.php

```diff
@@ -858,6 +860,16 @@ public function isChanged()
         return false;
     }
 
+    public function isEmptyAt($index)
```

Comment:
no data attached to token. currently, it means `new Token('')` (here and on 2.2-master line)
we cannot make `$tokens[$index] = null` or `unset($tokens[$index])` easy, as (beside BC breaker) it would require change for all collection looping...